src/meta/model/migration/src/m20230908_072257_init.rs

```diff
@@ -378,7 +378,7 @@ impl MigrationTrait for Migration {
                     )
                     .col(
                         ColumnDef::new(Fragment::StreamNode)
-                            .blob(BlobSize::Long)
+                            .blob()
```

Comment:
It's the result of  breaking changes from `sea-query`: https://github.com/SeaQL/sea-query/blob/master/CHANGELOG.md#new-features-1.

cc @yezizp2012 wonder if it's safe to change the migration scripts.

Reply:
The blob size is required by mysql backend. `MySQL` has four `BLOB` types: `TINYBLOB`, `BLOB`, `MEDIUMBLOB`, and `LONGBLOB`, every type has a corresponding length limit. After the breaking changes of `sea-query`, I think we need to specify the column type as `.custom(MySqlType::LongBlob)` if the backend is `MySQL`. 

FYI: both `PostgreSQL` and `SQLite` have a 1GB size limit for the `BLOB` type.

Reply:
Updated it https://github.com/risingwavelabs/risingwave/pull/19145/commits/77707e1b4f5924b0ed9c447ada8f38d2e24d3c97

Reply:
Done in #19207